### PR TITLE
Some suggestions and corrections of typos

### PR DESCRIPTION
**Pull request by roggenk**
Hope this helps...

Regards,
Kurt Roggen [BE]

### CODE REVIEW

File: virtualization/hyperv_on_windows/user_guide/export_import.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ To import a virtual machine using PowerShell, use the **Import-VM** command.  Th
 To complete an in place import of a virtual machine, the command would look similar to this. Recall that an in place import uses the files where they are stored at the time of import and retains the virtual machines id.
 
 ```powershell
-Import-VM -Path 'C:\<emport path>\2B91FEB3-F1E0-4FFF-B8BE-29CED892A95A.vmcx'
```
Comment (scooley) [inline virtualization/hyperv_on_windows/user_guide/export_import.md]: import

File: virtualization/hyperv_on_windows/user_guide/vmsession.md [hunk 2]
```diff
@@ -143,15 +143,15 @@ By the same token, sessions hold state.  Since persistent sessions persist, any
   
 3. Copy a file into the virtual machine.
   
-  To move `C:\host_path\data.txt` to the virtual machine from the host machine, run:
+  To copy `C:\host_path\data.txt` to the virtual machine from the host machine, run:
```
Comment (scooley) [inline virtualization/hyperv_on_windows/user_guide/vmsession.md]: Good catch :)

**Comment by scooley** [comment]
Good chances :).

File: virtualization/hyperv_on_windows/user_guide/export_import.md (hunk 1 above)
Comment (roggenk) [inline virtualization/hyperv_on_windows/user_guide/export_import.md]: vm export - see other Examples line 96 and 108

**Pull request merged**